assembly/mapr6.1.0/src/main/resources/examples/conf/hbase-env.secure.sh

```diff
@@ -0,0 +1,236 @@
+#
```

Comment:
do we want these files to be stored as 'examples' ? I think of them more as 'templates', since we copy in one or the other of these files into the rpm at build time. I wonder if the directory name will be confusing to future maintainers.

Reply:
Moved them all into a scripts folder and separated them out based on service. I don't think change between platforms (other than past installation methods). Maybe mapr should be the first unified assembly platform?